tests/api.suite.yml

```diff
@@ -9,4 +9,7 @@ modules:
     - Helper\Api
     - \Helper\Integration
     - REST:
-        depends: Phalcon
+        depends: PhpBrowser
+        url: 'http://%CODECEPTION_URL%:%CODECEPTION_PORT%'
```

Comment:
As I can see `CODECEPTION_*` variables is undefined for now.

Reply:
Specified as action point in #23